diff --git a/repos/spack_repo/builtin/packages/kokkos_nvcc_wrapper/package.py b/repos/spack_repo/builtin/packages/kokkos_nvcc_wrapper/package.py
index ba820d059b..7d7a6b3db1 100644
--- a/repos/spack_repo/builtin/packages/kokkos_nvcc_wrapper/package.py
+++ b/repos/spack_repo/builtin/packages/kokkos_nvcc_wrapper/package.py
@@ -3,8 +3,6 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 import os
 
-from spack_repo.builtin.build_systems.generic import Package
-
 from spack.package import *
 
 
@@ -168,9 +166,7 @@ class KokkosNvccWrapper(Package):
         mkdir(prefix.bin)
         install(src, prefix.bin)
 
-    def setup_dependent_build_environment(
-        self, env: EnvironmentModifications, dependent_spec: Spec
-    ) -> None:
+    def setup_dependent_build_environment(self, env, dependent_spec):
         wrapper = join_path(self.prefix.bin, "nvcc_wrapper")
         env.set("CUDA_ROOT", dependent_spec["cuda"].prefix)
         env.set("NVCC_WRAPPER_DEFAULT_COMPILER", self.compiler.cxx)
