diff --git a/repos/spack_repo/builtin/packages/kokkos_tools/package.py b/repos/spack_repo/builtin/packages/kokkos_tools/package.py
index 88ae81f8a9..c27751b3df 100644
--- a/repos/spack_repo/builtin/packages/kokkos_tools/package.py
+++ b/repos/spack_repo/builtin/packages/kokkos_tools/package.py
@@ -2,8 +2,6 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack_repo.builtin.build_systems.cmake import CMakePackage
-
 from spack.package import *
 
 
